sysdump/sysdump.go

```diff
@@ -824,6 +842,51 @@ func (c *Collector) submitLogsTasks(ctx context.Context, pods []*corev1.Pod, sin
 	return nil
 }
 
+func (c *Collector) submitFlavorSpecificTasks(ctx context.Context, f k8s.Flavor, path func(string) string) error {
+	switch f.Kind {
+	case k8s.KindEKS:
+		if err := c.pool.Submit(awsNodeDaemonSetName, func(ctx context.Context) error {
+			// Collect the 'kube-system/aws-node' DaemonSet.
+			d, err := c.client.GetDaemonSet(ctx, awsNodeDaemonSetNamespace, awsNodeDaemonSetName, metav1.GetOptions{})
+			if err != nil {
+				if errors.IsNotFound(err) {
+					c.logWarn("DaemonSet %q not found in namespace %q - this is expected when running in ENI mode", awsNodeDaemonSetName, awsNodeDaemonSetNamespace)
```

Comment:
If it is expected why are we telling that to the user in a form of a warning message? Maybe we could not print the message at all

Reply:
Makes sense — I've changed it to `logDebug` as it may still be useful. WDYT?